debian/ceph-common.postrm

```diff
@@ -24,8 +24,9 @@ case "$1" in
     ;;
 
     purge)
-	rm -rf /var/log/ceph 
+	rm -rf /var/log/ceph
 	rm -rf /etc/ceph
+	usermod -L -e 1 ceph
```

Comment:
likewise, please put something like
```sh
    rm -rf 
    if [ -f /etc/shadow ]; then
      usermod -L -e 1 $SERVER_USER
    else
      usermod -L $SERVER_USER
    fi
```
and you need to query `/etc/default/ceph` for the username of the `ceph` user. it is might not be `ceph` if user overrides it.
```sh
[ -f "/etc/default/ceph" ] && . /etc/default/ceph
[ -z "$SERVER_USER" ] && SERVER_USER=ceph
```